modules/client/src/connext.ts

```diff
@@ -831,7 +831,7 @@ export class ConnextClient implements IConnextClient {
       throw new Error(`No way to decode transfer, this should never happen!`);
     }
 
-    const cipher = EthCrypto.cipher.parse(encryptedPreImage);
+    const cipher = EthCrypto.parse(encryptedPreImage);
 
     const preImage = await EthCrypto.decryptWithPrivateKey(privateKey, cipher);
```

Comment:
Why generate this intermediate `cipher` var? If it's never used, then we should hide it w/in the EthCrypto module & simplify the `decryptWithPrivateKey` function sig accordingly